io/include/pcl/io/vlp_grabber.h

```diff
@@ -80,13 +80,34 @@ namespace pcl
       virtual std::string
       getName () const;
 
+      /** \brief Allows one to customize the colors used for each of the lasers. */
```

Comment:
Method description similar to the one before. Add doc regarding input parameters.